src/prog_algs/state_estimators/particle_filter.py

```diff
@@ -29,21 +28,25 @@ class ParticleFilter(state_estimator.StateEstimator):
          * x0_uncertainty : Initial uncertainty in state e.g., 0.5
          * R (Number) : Measurement Noise. e.g., 0.1
     """
-    default_parameters = {
-            'n': 0.1, # Sensor Noise
-            'num_particles': 20, 
-            'resample_fcn': residual_resample, # Resampling function ([weights]) -> [indexes]
-            'x0_uncertainty': 0.5   # Initial State Uncertainty
-                                    # Can be:
-                                    #   1. scalar (standard deviation applied to all),
-                                    #   2. dict (stardard deviation for each)
-        }
 
     def __init__(self, model, x0, measurement_eqn = None, **kwargs):
-        super().__init__(model, x0, measurement_eqn = measurement_eqn, **kwargs)
 
+        self.default_parameters = {
```

Comment:
From my understanding we moved the default_parameters into the instance because we wanted the default parameters  of 'n' and 'x0_uncertainty' to be the process noise for the particle filter automatically rather than having it as the same value for all the states.

Reply:
Follow up. I think it's fine if we move default parameters back to a property of the class. I just think it adds some unnecessary complexity having to specify the process noise parameters explicitly because the default for the particle filter should be the model process noise. 
Spencer

Reply:
You can disregard these comments. Getting rid of 'n' and moving the default parameters up to a property of the class fixes the problem we ran into.